packages/datetime/src/blueprint-datetime.scss

```diff
@@ -2,10 +2,8 @@
 // Licensed under the Apache License, Version 2.0.
 
 @import "common/react-day-picker-overrides";
-@import "components/date-input/dateinput";
-@import "components/date-picker/datepicker";
+@import "components/date-input/date-input";
 @import "components/date-picker/date-picker";
 @import "components/date-picker/date-picker-caption";
-@import "components/date-range-picker/daterangepicker";
 @import "components/date-range-picker/date-range-picker";
-@import "components/time-picker/timepicker";
+@import "components/time-picker/time-picker";
```

Comment:
No changes to `date-input` or `time-picker`, these were simply renamed for consistency.